app/models/health_care_application.rb

```diff
@@ -261,7 +261,7 @@ def log_async_submission_failure
   end
 
   def log_submission_failure_details
-    return if form.blank?
+    return if parsed_form.blank?
```

Comment:
This checks the `parsed_form` now since that is where we are getting our values for the logs from.